<?xml version="1.0"?>

<launch>
  
  <node pkg="hector_stability_assistance" type="stability_visualization_node" name="stability_visualization" output="screen" respawn="true">
    <param name="update_frequency" value="20"/>
    <param name="elevation_layer_name" value="elevation"/>
    
    <remap from="elevation_map" to="/elevation_mapping/elevation_map_raw"/>
  </node>

 
</launch>
